content/verein/constitution.fr.md

```diff
@@ -0,0 +1,181 @@
+
+a) soutenir le grand public dans l'exercice de son droit à la vie privée (« droit à la liberté d'information ») en l'informant et en le conseillant sur toutes les questions relatives à la protection des données à caractère personnel (en tant que droit fondamental de l'homme conformément à l'article 8, paragraphe 1, de la charte des droits fondamentaux de la l'Union européenne) et faire ainsi progresser le gouvernement démocratique dans l'espace numérique ainsi que l'éducation du grand public, et  
+b) fournir des informations sur les pratiques de protection des données dans les entreprises et autres organisations afin de protéger les données personnelles des consommateurs et, pour ce faire, encourager les actions de conseil et de protection des consommateurs en ce qui concerne les réseaux numériques.
+
+**2.2** Ces objectifs sont atteints en particulier grâce à :
+
+a) la mise en place et le développement d'un générateur de demandes conformément aux articles 15 et suivants du RGPD et ainsi offrir des lettres types pour de telles demandes afin de soutenir le grand public dans l'exercice de son droit à la vie privée,  
+b) la fourniture et la gestion de matériel d'information (articles, guides, rapports, brochures, magazines imprimés et vidéos) pour la formation, la science et la recherche, en informant et en conseillant le grand public sur des sujets tels que les conséquences de l'utilisation des réseaux numériques sur leur vie privée, les droits et possibilités offerts par le droit fondamental à la protection des données et le RGPD en particulier, ainsi que sur la manière d'utiliser ces droits,
+c) la gestion d'une base de données sous licence libre contenant des informations sur la manière de contacter les entreprises, les organisations gouvernementales et autres pour des demandes liées à la protection de la vie privée, ainsi que des informations sur les catégories potentielles de données stockées par ces entreprises ; en outre, la modération des contributions des utilisateurs à cette base de données afin de permettre l'échange d'informations dans l'intérêt de la protection des consommateurs,  
+d) la sensibilisation du public et la réalisation de campagnes, ainsi que l'organisation ou la participation à des conférences, séminaires et autres événements destinés à conseiller le grand public sur le droit à la vie privée et à faire avancer le débat concernant un droit fondamental à la liberté d'information, et  
+e) la gestion et le développement des plates-formes en ligne ouvertes, gratuites et accessibles au public datenanfragen.de (en allemand), datarequests.org (en anglais), demandetesdonnees.fr (en français) et éventuellement dans d'autres langues, qui permettent d'accéder au générateur décrit au point a) et à la base de données décrite au point c), de rendre le matériel d'information décrit au point b) accessible au public et, enfin, de faire de la publicité pour la sensibilisation du public décrite au point d).
```

Comment:
Two small remarks here:

* The 'd' in 'Datenanfragen.de' should be capitalized.
* Usually, you are absolutely right with including demandetesdonnees.fr in these lists. In this case however, as this is a translation of an official legal document which doesn't mention dtd.fr, I would rather leave it out.

Reply:
Even as an URL? I understand it should be as the association name but generally URLs should not contain capital letters.
That's okay for the second point :wink:

Reply:
I agree that URLs don't usually contain capital letters but in this case this *is* the name of the website that just happens to look like (and be the same) as the website URL. :D That's just the 'style guide' we have defined.

If however we were to write an actual URL that isn't just the name (like www.datenanfragen.de/generator), the 'd' would indeed be lower-case. Although this usually only happens in print documents. On the web, we of course use actual links.

Reply:
Understood 👍